docs/knowledge_base/policy_development.md

```diff
@@ -1,3 +1,33 @@
+# Policy Basics
+
+This is a basic policy using an `issue` rule to block any HIGH/CRITICAL issues.
+
+```rego
+# METADATA
+# title: Limit risk
+# description: |
+#   Block issues based on risk level.
+package policy.v1
+
+import data.phylum.level
+import rego.v1
+
+# METADATA
+# title: risk level cannot exceed medium
+deny contains issue if {
+    some issue in data.issues
+    issue.severity > level.MEDIUM
+}
+```
+
+The `title` and `description` from the initial metadata comments are displayed in the Phylum UI and are highly recommended.
+
+The `package policy.v1` line must be present. This is how OPA finds the policy's rules.
```

Comment:
This page references OPA as `opa`, `OPA`, and "OPA" (no formatting). Suggest standardizing on one and being consistent.

This is also the first time OPA is used in this document. It should be spelled out fully, provided as an acronym, and ideally linked to a stable resource describing it in more detail. Even if this was done on a different page, I think it is still worth doing again on this page since this page is likely to be referenced and folks may not read the surrounding content.

Reply:
I believe this is addressed. I included the acronym with a link on first occurrence and normalized the usage of OPA to what I thought looked correct. I did leave some `opa` when I felt like it was directly referencing the OPA CLI tool and not the larger OPA framework.

Reply:
The [commit where the change was made](https://github.com/phylum-dev/documentation/pull/194/commits/9832ae0ada30cdc85df170b1d722365310758194) might need to be updated again. It appears there that a number of instances of `opa` were removed instead of updated. That results in text like `` (empty format) or ` other text here` (blank space at the beginning of formatted text where it used to begin with `opa`.

Reply:
Strange. Must have been user error with my "find." I think I fixed them all now.